cluster/cluster.go

```diff
@@ -0,0 +1,113 @@
+// Package cluster is a Test Cluster Framework -- see the motivating
```

Comment:
You need to include the license before the blob.

Reply:
✅